frontend/src/core/composables/useEventStream.ts

```
@@ -60,83 +60,109 @@ async function readSseStream(
+      })
       status.value = 'closed'
     } catch (err) {
+      if (abortController !== controller) return // stale invocation, ignore
       if (err instanceof DOMException && err.name === 'AbortError') {
         status.value = 'closed'
       } else {
         error.value = err instanceof Error ? err : new Error(String(err))
         status.value = 'closed'
       }
     } finally {
+      if (abortController !== controller) return // stale invocation, ignore
```

Biome (2.5.0)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)